ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -1315,16 +1307,24 @@ odla_value odla_Gemm(odla_value lhs, odla_bool transpose_lhs, odla_value rhs,
   dnnl::matmul::desc md(lhs_md, rhs_md, ret_md);
   dnnl::matmul::primitive_desc pd(md, g_comp->eng);
   dnnl::primitive prim = dnnl::matmul(pd);
-
-  add_op(prim, {{DNNL_ARG_SRC, lhs->mem},
-                {DNNL_ARG_WEIGHTS, rhs->mem},
-                {DNNL_ARG_DST, ret_mem}});
+  if (bias) {
+    dnnl::memory::desc bias_md({1, N}, dt, dnnl::memory::format_tag::ab);
+    auto bias_mem = dnnl::memory(bias_md, g_comp->eng);
+    add_op(prim, {{DNNL_ARG_SRC, lhs->mem},
+                  {DNNL_ARG_WEIGHTS, rhs->mem},
+                  {DNNL_ARG_BIAS, bias_mem},
+                  {DNNL_ARG_DST, ret_mem}});
+  } else {
+    add_op(prim, {{DNNL_ARG_SRC, lhs->mem},
+                  {DNNL_ARG_WEIGHTS, rhs->mem},
+                  {DNNL_ARG_DST, ret_mem}});
+  }
 
   odla_value v = CreateValue(ret_mem, output_dims, bias ? nullptr : id);
 
   InterpretIfNeeded();
 
-  return bias ? odla_Add(v, bias, id) : v; // FIXME:
+  return v;
```

Comment:
Please change line 1323 to 
```
odla_value v = CreateValue(ret_mem, output_dims, id)
```
otherwise, if there is bias, id won't be created.
(In original code, the id will be passed to "odla_Add(..., id)")